### DISCUSSION TITLE
gum spin in Windows

### DISCUSSION THREAD

**Discussion by Gianlucas94** [charmbracelet/gum#421] [General]
Hi there!
Someone managed to run gum spin in Windows?

When I try to run the code bellow nothing happens.

`gum spin --spinner dot --title "Buying Bubble Gum..." -- sleep 5`

**Comment by gitrust** [comment]
I looked for the same solution.

Since there is no `sleep` in windows, I tried `timeout` with no luck.

Then I created a `sleep.bat`  and put following command into it. This batch file needs to be on PATH to be usable

```
@echo off
ping -n %1 127.0.0.1 > nul
```

After that I could run that command like a charm

```
gum spin --spinner dot --title "Buying Bubble Gum..." -- sleep 5
```
